src/main/resources/templates/db_scripts/schema.json

```diff
@@ -594,6 +594,17 @@
         }
       ]
     },
+    {
+      "tableName": "holdings_records_source",
+      "withMetadata": false,
```

Comment:
Why would we not want the change metadata for these records?

Reply:
I don't know what that property does and that code change predates me. I'm happy to change its value if you're telling me it needs to be changed.

Reply:
Yes please, I believe it is needed in order to present the `last updated` column in the settings UI.